### PR TITLE
Commit for issue#39

### PR DESCRIPTION
**Pull request by adhyay2000**
Added the server-side validation.
If a client is new joinee,he can create a new chatroom by sending NEW command
or validate by CHKUSR command. The chatroom are generated at a single point,hence
the disperancy coming in earlier case is resolved.

### CODE REVIEW

**Comment by adhyay2000** [comment]
fix for issue #39.

File: cnew.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 sess = PromptSession()
 sepr = chr(969696)
+websocket = None
```
Comment (t0xic0der) [inline cnew.py:12]: Please avoid such cross-type object initialization as far as possible.

File: cnew.py [hunk 2]
```diff
@@ -29,7 +30,7 @@ def decrjson(self, data):
         return self.suit.decrypt(data.encode("utf8")).decode("utf8")
 
 
-async def consumer_handler(cphrsuit, websocket, username, chatroom, servaddr):
+async def consumer_handler(cphrsuit, username, chatroom, servaddr):
```
Comment (t0xic0der) [inline cnew.py:33]: The websocket object must be passed here as a parameter.

File: cnew.py [hunk 3]
```diff
@@ -46,7 +47,7 @@ async def consumer_handler(cphrsuit, websocket, username, chatroom, servaddr):
             pass
 
 
-async def producer_handler(cphrsuit, websocket, username, chatroom, servaddr):
+async def producer_handler(cphrsuit, username, chatroom, servaddr):
```
Comment (t0xic0der) [inline cnew.py:50]: The websocket object must be passed here as a parameter.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
A lot of changes hail from #84. Please look at the suggestions I have provided there and work accordingly.

**Comment by adhyay2000** [comment]
Hi, I have added my justification for using this way, there. Please look that and reply what I should do?

**Pull request closed without merging**